src/lib_param.F90

```diff
@@ -137,6 +137,9 @@ module lib_param
    real(dp) :: g_spin            ! spin degeneracy
    real(dp) :: delta             ! delta for G.F. 
    real(dp) :: dos_delta         ! additional delta to force more broadening in the DOS 
+   integer  :: deltaModel        ! Used for phonon G.F. ! delta**2, 2*delta*w, Mingo's
+   real(dp) :: wmax              ! Maximum frequency in Mingo's model
```

Comment:
Can we initialize `deltaModel=0, wmax=0.d0` here or in lib_params.F90::set_defaults?